package.json

```diff
@@ -13,14 +13,13 @@
     "test": "yarn test:e2e && yarn test:workflow",
     "test:e2e": "yarn test:e2e:codegen && jest --passWithNoTests --runInBand --verbose",
     "test:e2e:codegen": "npx polywrap app codegen -m ./src/__tests__/types/polywrap.app.yaml -g ./src/__tests__/types/wrap",
-    "test:workflow:mock": "npx polywrap run ./workflows/e2e.yaml -c ./workflows/client-config.ts -v ./workflows/validator.mock.cue",
-    "test:workflow": "npx polywrap run ./workflows/e2e.yaml -v ./workflows/validator.cue"
+    "test:workflow:mock": "npx polywrap run -c ./workflows/client-config.ts -o test.json -q",
+    "test:workflow": "npx polywrap run polywrap.test.yaml -v ./workflows/validator.cue"
```

Comment:
The workflow fails. Is that an issue here?

Also, the `-v` option is no longer supported. It is preventing the workflow from running.

Reply:
Should the `test` script be using `test:workflow:mock` instead of `test:workflow`?

Reply:
Oops I forgot to fix the workflow test commands

Reply:
Nice catch~